internal/stack/resources.go

```diff
@@ -163,7 +163,7 @@ func applyResources(profile *profile.Profile, stackVersion string) error {
 		"apm_enabled":          profile.Config(configAPMEnabled, "false"),
 		"geoip_dir":            profile.Config(configGeoIPDir, "./ingest-geoip"),
 		"kibana_http2_enabled": profile.Config(configKibanaHTTP2Enabled, "true"),
-		"logsdb_enabled":       profile.Config(configLogsDBEnabled, "false"),
+		"logsdb_enabled":       profile.Config(configLogsDBEnabled, "true"),
```

Comment:
Change in default value included here only for testing, to be removed before merging, and eventually added in a different PR.